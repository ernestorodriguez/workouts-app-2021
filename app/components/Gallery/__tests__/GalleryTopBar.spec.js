import React from 'react';
import { shallow } from 'enzyme';
import TopBar from '../GalleryTopBar';

describe('TopBar component', () => {
  test('should match snapshot', () => {
    const config = {
      startDateChangeHandler() { },
      categoryChangeHandler() { },
      startDateSelector: {},
      availableCategories: [],
    };
    // eslint-disable-next-line react/jsx-props-no-spreading
    const wrapper = shallow(<TopBar {...config} />);
    expect(wrapper).toMatchSnapshot();
  });
});
