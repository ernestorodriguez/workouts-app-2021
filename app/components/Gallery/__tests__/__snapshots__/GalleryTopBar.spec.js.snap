// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar component should match snapshot 1`] = `
<header
  className="gallery-header"
  data-js="top-bar"
>
  <Dropdown
    additionLabel="Add "
    additionPosition="top"
    className="gallery-header__dropdown start-date-selector"
    closeOnBlur={true}
    closeOnEscape={true}
    deburr={false}
    fluid={true}
    icon="dropdown"
    minCharacters={1}
    noResultsMessage="No results found."
    onChange={[Function]}
    openOnFocus={true}
    options={Array []}
    placeholder="Select Month"
    renderLabel={[Function]}
    searchInput="text"
    selectOnBlur={true}
    selectOnNavigation={true}
    selection={true}
    wrapSelection={true}
  />
  <Dropdown
    additionLabel="Add "
    additionPosition="top"
    className="gallery-header__dropdown category-selector"
    closeOnBlur={true}
    closeOnEscape={true}
    data-js="start-date-filter"
    deburr={false}
    fluid={true}
    icon="dropdown"
    minCharacters={1}
    multiple={true}
    noResultsMessage="No results found."
    onChange={[Function]}
    openOnFocus={true}
    options={Array []}
    placeholder="Select Categories"
    renderLabel={[Function]}
    searchInput="text"
    selectOnBlur={true}
    selectOnNavigation={true}
    selection={true}
    wrapSelection={true}
  />
</header>
`;
